### DISCUSSION TITLE
How can I access cached data in AdminPanelProvider

### DISCUSSION THREAD

**Discussion by alkoumi** [filamentphp/filament#9679] [Help] [answered]
### Then I want to access cached `$setting variable` in AdminPanelProvider 

```
return $panel
            ->id('go')
            ->path('/go')
            ->default()
            ->brandName($setting->brandName)
            ->brandLogo($setting->brandLogo)
```

### if I use cache directly Cache::get() in AdminPanelProvider its Thrown an error 

``` Target class [cache] does not exist.```

How I can access cached `$setting variable` in AdminPanelProvider 

Now I am using ```DB::table('settings')``` But I need to make it faster same like Cache form redis

Thanks for Helping

**Comment by leandrocfe** [comment]
not sure
```php
$panel
->bootUsing(function (Panel $panel) {
    //access here
})
```

**Comment by sinnbeck** [comment, marked as answer]
```php
  ->brandName(fn() => Cache::get('brandName'))
